Merge Remdesivir info from #75 and #111 (#124)
greenelab/covid19-review: content/20.treatments.md

##### Remdesivir
### Antiviral Approach: Remdesivir
Remdesivir (GS-5734) was developed by Gilead Sciences to treat Ebola and was briefly tested in the clinic.
It does not have any FDA approved use.
Although a clinical trial in the Democratic Republic of Congo found some evidence of effectiveness against ebola, two antibody preparations were found to be more effective, and Remdesivir was not pursued [@doi:10.1056/NEJMoa1910993]. 
Remdesivir (Gilead) is an adenosine analogue that, once incorporated into the nascent viral chain, causes premature termination of transcription [@doi:10.1038/s41422-020-0282-0].

### Anticipated Mechanism
Remdesivir is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair causing chain termination [@doi:10.1074/jbc.AC120.013056; @doi:10.1128/mBio.00221-18].
Although it was developed against Ebola, it also inhibits the MERS-CoV and SARS-CoV polymerase and inhibits coronavirus replication in cell culture assays with submicromolar IC50s [@doi:10.1126/scitranslmed.aal3653].
It also inhibits SARS-CoV-2, showing synergy with chloroquine [@doi:10.1038/s41422-020-0282-0].

### Current Evidence

> Previous work has shown Remdesivir to be an effective treatment for viral pathogens such as SARS-CoV and MERS-CoV in cultured cells and animal models, and a recent study found that administration of Remdesivir to non-human primate models resulted in 100% protection against infection by the Ebola virus.

Author: ```suggestion
In addition to the previous work showing Remdesivir to be an effective treatment for viral pathogens such as SARS-CoV and MERS-CoV in cultured cells and animal models, a recent study found that administration of Remdesivir to non-human primate models resulted in 100% protection against infection by the Ebola virus.
```

All were in late stages of COVID-19 infection, and these reports are inconclusive about the drug's efficacy.
Remdesivir recently entered controlled clinical trials, and as of March 2020, there are six clinical trials underway to treat COVID-19 patients at both early and late stages of infection and in combinations with other drugs [@doi: 10.1038/s41422-020-0282-0; @url:https://clinicaltrials.gov/ct2/show/NCT04292730; @url:https://clinicaltrials.gov/ct2/show/NCT04292899; @url:https://clinicaltrials.gov/ct2/show/NCT04280705; @url:https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR; @url:https://clinicaltrials.gov/ct2/show/NCT04252664; @url:https://clinicaltrials.gov/ct2/show/NCT04257656].

### Summary
Remdesivir is a major drug candidate since it attacks the virus with high potency and known mechanism.
Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.
Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.

> Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas Covid-19 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against Covid-19.

Author: ```suggestion
Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas SARS-CoV-2 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against SARS-CoV-2.
```

Summarize the state of the antiviral approach.

##### Remdesivir
### Antiviral Approach: Remdesivir

> Remdesivir (GS-5734) was developed by Gilead Sciences to treat of Ebola and was briefly tested in the clinic.  It does not have any FDA approved use.  There was some evidence of effectiveness, but it was not as effective as two antibody preparations so it was not pursued. 

Author: ```suggestion
Remdesivir (GS-5734) was developed by Gilead Sciences to treat of Ebola and was briefly tested in the clinic.
It does not have any FDA approved use.
There was some evidence of effectiveness, but it was not as effective as two antibody preparations so it was not pursued [@doi:10.1056/NEJMoa1910993]. 
```

Summarize the state of the antiviral approach.

##### Remdesivir
### Antiviral Approach: Remdesivir
Remdesivir (GS-5734) was developed by Gilead Sciences to treat of Ebola and was briefly tested in the clinic.
It does not have any FDA approved use.

> There was some evidence of effectiveness, but it was not as effective as two antibody preparations so it was not pursued [@doi:10.1056/NEJMoa1910993]. 

Author: ```suggestion
Although a clinical trial in the Democratic Republic of Congo found some evidence of effectiveness against ebola, two antibody preparations were found to be more effective, and Remdesivir was not pursued [@doi:10.1056/NEJMoa1910993]. 
```

### Current Evidence
Remdesivir was used on four Covid-19 patients under compassionate use guidelines. All were in late stages of Covid-19 infection and the results have been mixed.  More recently Remdesivir entered controlled clinical trials.  As of March 2020, there are six clinical trials underway to treat Covid-19 patients at both early and late stages of infection and in combinations with other drugs (https://www.nature.com/articles/s41422-020-0282-0). (NCT04292730, NCT04292899, NCT04280705, 2020-000936-23, NCT04252664, NCT04257656). 
Previous works already showed that Remdesivir was effective in the treatment of Covid-19 similar viral pathogens, such as SARS-Cov and MERS-Cov, in animal models, while a recent study showed that Remdesivir resulted in 100% protection against the Ebola virus in human liver Cancer cell line, which is also sensitive to the novel Corona virus [@doi:10.1038/s41422-020-0282-0]. 

> Upon FDA approval Gilead enrolled about a 1000 patients from COVID-19 hit countries for two phase III clinical trials to test the efficacy of the drug in adults. 

Author: ```suggestion
```

> Specifically, one trial will test 400 patients with severe COVID-19 symptoms, while the second trial will randomise the remaining amount of patients with mild symptoms. 

Author: ```suggestion 
```

> Specifically, one trial will test 400 patients with severe COVID-19 symptoms, while the second trial will randomise the remaining amount of patients with mild symptoms. 

Author: ```suggestion 
```

> In both studies, patients will receive the same Remdesivir dose for 10 days (Gilead.com). 

Author: ```suggestion
```

> Remdesivir is also currently involved in a trial to test the drug against Ebola virus since 2015. 

Author: ```suggestion
```

> However, in August 2019, after a large phase III trial in the Democratic Republic of Congo, the drug was declared to be less effective compared to other monoclonal antibody in use (Biopharmadive.com). 

Author: ```suggestion 
```

### Summary

> Remdesivir is a major drug candidate since it attacks the virus with high potency and known mechanism.  Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.  Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.  Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas Covid-19 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against Covid-19.    In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens. They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.   

Author: ```suggestion
Remdesivir is a major drug candidate since it attacks the virus with high potency and known mechanism.
Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.
Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.
Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas Covid-19 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against Covid-19.
In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens.
They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.   
```

Summarize the state of the neutralizing antibody approach.

### Antiviral Approach: Remdesivir

> Remdesivir (GS-5734) was developed by Gilead Sciences to treat of Ebola and was briefly tested in the clinic.  It does not have any FDA approved use.  There was some evidence of effectiveness, but it was not as effective as two antibody preparations so it was not pursued. 

Author: ```suggestion
Remdesivir (GS-5734) was developed by Gilead Sciences to treat of Ebola and was briefly tested in the clinic.
It does not have any FDA approved use.
There was some evidence of effectiveness, but it was not as effective as two antibody preparations so it was not pursued [@doi:10.1056/NEJMoa1910993].
```

> https://www.sciencemag.org/news/2019/08/finally-some-good-news-about-ebola-two-new-treatments-dramatically-lower-death-rate

Author: ```suggestion
```

   
### Anticipated Mechanism

> Remdesivir is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair causing chain termination (https://www.jbc.org/content/early/2020/02/24/jbc.AC120.013056.abstract)(https://mbio.asm.org/content/9/2/e00221-18).   Although it was developed against Ebola, it also inhibits the MERS CoV and SARS CoV polymerase and inhibits corona virus replication in cell culture assays with submicromolar IC50s (https://stm.sciencemag.org/content/9/396/eaal36530.  It also inhibits CoV-19 showing synergy with chloroquine.

Author: ```suggestion
Remdesivir is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair causing chain termination [@doi:10.1074/jbc.AC120.013056; @doi:10.1128/mBio.00221-18].   
Although it was developed against Ebola, it also inhibits the MERS CoV and SARS CoV polymerase and inhibits coronavirus replication in cell culture assays with submicromolar IC50s [@doi:10.1126/scitranslmed.aal3653].
It also inhibits SARS-CoV-2, showing synergy with chloroquine [@doi:10.1038/s41422-020-0282-0].
```

### Current Evidence

> Remdesivir was used on four Covid-19 patients under compassionate use guidelines. All were in late stages of Covid-19 infection and the results have been mixed.  More recently Remdesivir entered controlled clinical trials.  As of March 2020, there are six clinical trials underway to treat Covid-19 patients at both early and late stages of infection and in combinations with other drugs (https://www.nature.com/articles/s41422-020-0282-0). (NCT04292730, NCT04292899, NCT04280705, 2020-000936-23, NCT04252664, NCT04257656). 

Author: ```suggestion
Remdesivir was used on four Covid-19 patients under compassionate use guidelines.
All were in late stages of Covid-19 infection and the results have been mixed.
More recently Remdesivir entered controlled clinical trials.
As of March 2020, there are six clinical trials underway to treat Covid-19 patients at both early and late stages of infection and in combinations with other drugs [@doi: 10.1038/s41422-020-0282-0; @url:https://clinicaltrials.gov/ct2/show/NCT04292730; @url:https://clinicaltrials.gov/ct2/show/NCT04292899; @url:https://clinicaltrials.gov/ct2/show/NCT04280705; @url:https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR; @url:https://clinicaltrials.gov/ct2/show/NCT04252664; @url:https://clinicaltrials.gov/ct2/show/NCT04257656].
```

### Summary

> Remdesivir is a major drug candidate since it attacks the virus with high potency and known mechanism.  Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.  Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.  Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas Covid-19 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against Covid-19.    In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens. They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.   

Author: ```suggestion
Remdesivir is a major drug candidate since it attacks the virus with high potency and known mechanism.  
Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.  
Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.  
Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas SARS-CoV-2 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against SARS-CoV-2.    
In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens. 
They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.   
```
